BeVolt/Apps/Src/main.c

```diff
@@ -72,6 +72,8 @@ int main(){
 			break;
 		}
 
+		// TODO: Implement heartbeat for RUN light at a visible frequency
```

Comment:
Can we make this an issue if it's not already? @ClarkPoon

Reply:
Good catch, I just created a new issue for this